{{cookiecutter.project_slug}}/config/settings/local.py

```diff
@@ -83,6 +83,15 @@
         # The node container isn't started (yet?)
         pass
     {%- endif %}
+{% if cookiecutter.windows == 'y' %}
+# RunServerPlus
+# ------------------------------------------------------------------------------
+# After how many seconds auto-reload should scan for updates in poller-mode
+RUNSERVERPLUS_POLLER_RELOADER_INTERVAL = 5
```

Comment:
5 seconds seems a bit long, no? The default value of 1 second seems more appropriate:

```suggestion
RUNSERVERPLUS_POLLER_RELOADER_INTERVAL = 1
```

Actually, since it's the default interval, do we need to define it? I think we could omit it...